corpora-api.yml

```diff
@@ -493,6 +497,131 @@ components:
     visibility:
       type: string
       enum: [public,private]
+    project:
+      type: object
+        properties:
+          description:
+            type: string
+          name:
+            type: string
+          id:
+            $ref: '#/components/schemas/project_uuid'
+          s3_bucket_key:
+            type: string
+          submitter:
+            type: object
+            properties:
+              id:
+                $ref: '#/components/schemas/user_uuid'
+              email:
+                type: string
+          links:
+            type: object
+            properties:
+              protocol:
+                type: array
+                items:
+                  type: string
+              other:
+                type: array
+                items:
+                  type: string
+              raw_data:
+                type: array
+                items:
+                  type: string
+              attestation:
+                type: array
+                items:
+                  type: string
+          datasets:
```

Comment:
Seems worthwhile to split out `dataset` into its own separate object in case sometime down the line, we need to return the dataset object for an API call instead of only project.

Reply:
I didn't want to prematurely optimize, but it can be broken out.